src/database/schema/ChatSchema.ts

```diff
@@ -355,8 +405,10 @@ class ChatSchema implements BaseDbSchema {
         response?.message?.updated_at,
         JSON.stringify(response),
         response?.message?.id,
+        JSON.stringify(response?.message?.attachments),
         this.id
       ];
+      console.warn('updateReplacement', JSON.stringify(updateReplacement));
 
       await db.executeSql(updateQuery, updateReplacement);
     } catch (e) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [399-410]

The SQL query has been updated to include `attachment_json`, but there's no check if `attachment_json` exists or not before updating the database. This could potentially lead to errors.



<!-- This is an auto-generated comment by OSS CodeRabbit -->